jumpscale/packages/tfgrid_solutions/chats/threebot.py

```diff
@@ -118,6 +118,7 @@ def domain_select(self):
         self.gateway = domains[self.domain]["gateway"]
         self.gateway_pool = domains[self.domain]["pool"]
         self.domain = f"{self.threebot_name}-{self.solution_name}.{self.domain}"
+        self.domain = j.sals.zos.gateway.correct_domain(self.domain)
```

Comment:
here also the self.domain was corrected but it was used before?

Reply:
same as above